tests/phpunit/CiviTest/CiviCaseTestCase.php

```diff
@@ -114,11 +96,48 @@ public function setUp(): void {
    */
   public function tearDown(): void {
     $this->customDirectories(['template_path' => FALSE]);
+    $this->tablesToTruncate = [
+      'civicrm_activity',
+      'civicrm_contact',
+      'civicrm_case',
+      'civicrm_case_contact',
+      'civicrm_case_activity',
+      'civicrm_case_type',
+      'civicrm_activity_contact',
+      'civicrm_managed',
+      'civicrm_relationship',
+      'civicrm_uf_match',
+      'civicrm_activity',
+      'civicrm_group_contact',
+      'civicrm_file',
+      'civicrm_entity_file',
+    ];
     $this->quickCleanup($this->tablesToTruncate, TRUE);
     CRM_Case_XMLRepository::singleton(TRUE);
     parent::tearDown();
   }
 
+  public static function setUpBeforeClass(): void {
+    parent::setUpBeforeClass();
+    CRM_Core_BAO_ConfigSetting::enableComponent('CiviCase');
```

Comment:
@eileenmcnaughton just wondering if we should be enabling the extension here instead of the component?

Reply:
@seamuslee001 let's consider that for a follow up - am fighting the hard coded values atm